integration/tests/iam_v2_no_legacy_to_v2_force_upgrade.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#shellcheck disable=SC2034
+test_name="iam_force_upgrade_to_v2_with_no_legacy"
+test_upgrades=true
+test_upgrade_strategy="none"
+test_deploy_inspec_profiles=(a2-iam-no-legacy-integration)
+test_upgrade_inspec_profiles=(a2-iam-no-legacy-integration a2-api-integration)
```

Comment:
Your purpose states that these changes verify two things:

_(a) The system will "not be disrupted" by the force-upgrade._
I am wondering if you could make this more concrete. Does "not disrupted" mean something like: it runs a standard battery of tests that all pass? If so, how do we know what constitutes that standard suite? Or does it mean something else?

_(b) No legacy policies will be present following the force-upgrade._
Which of the tests embedded in the controls above confirm this?

Reply:
`a2-api-integration` verifies that our APIs continue to behave as we expect. I'm actually going to swap that for `a2-deployment-integration`, which includes `a2-api-integration` and some system checks (as in, all the services are up and running, etc).

`a2-iam-no-legacy` verifies that permissions work as expected on fresh v2 install with no legacy policies. The key test here is `chef_managed_access.rb` which verifies permissions for all Chef-managed roles

Reply:
I've added some comments to this effect